<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rastreador de Salud</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutInput">
      <item>
       <widget class="QLabel" name="lblPeso">
        <property name="text">
         <string>Peso:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spPeso">
        <property name="suffix">
         <string> kg</string>
        </property>
        <property name="maximum">
         <double>300.0</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSis">
        <property name="text">
         <string>Sistólica:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spSis">
        <property name="suffix">
         <string> mmHg</string>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDia">
        <property name="text">
         <string>Diastólica:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spDia">
        <property name="suffix">
         <string> mmHg</string>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblGlu">
        <property name="text">
         <string>Glucosa:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spGlu">
        <property name="suffix">
         <string> mg/dL</string>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGuardar">
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutFilter">
      <item>
       <widget class="QLabel" name="lblIni">
        <property name="text">
         <string>Inicio:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="fechaIni">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFin">
        <property name="text">
         <string>Fin:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="fechaFin">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFiltrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tabla"/>
    </item>
    <item>
     <widget class="QLabel" name="lblStats">
      <property name="text">
       <string>Estadísticas se mostrarán aquí...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutActions">
      <item>
       <widget class="QPushButton" name="btnCalcular">
        <property name="text">
         <string>Calcular Estadísticas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExportar">
        <property name="text">
         <string>Exportar a CSV</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnLogout">
        <property name="text">
         <string>Cerrar Sesión</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
